10up/publisher-media-kit
# Enhancement/92 Bump WordPress and PHP minimums

### Description of the Change
Bumped the minimum WordPress and PHP minimums.
* WordPress to 5.7
* PHP to 7.4


Closes #92

### How to test the Change
Check if the plugin is working as expected.
Check the Github Actions working without fails.

### Changelog Entry
> Changed - Bump min WordPress to 5.7 and min PHP to 7.4

### Credits

Props @vikrampm1 @faisal-alvi @Sidsector9 


### Checklist:
